index.html

```diff
@@ -817,32 +723,26 @@ <h4>The <dfn>SemanticType</dfn> dictionary</h4>
       <section data-dfn-for="ThingActionInit">
         <h4>The <dfn>ThingActionInit</dfn> dictionary</h4>
         <pre class="idl">
-          dictionary ThingActionInit {
-            DOMString name;
-            ThingDescription inputDataDescription;
-            ThingDescription outputDataDescription;
-            sequence&lt;SemanticType&gt; semanticTypes;
-            Function action;
+          dictionary ThingActionInit: SemanticAnnotations {
+            required DOMString name;
+            ValueType inputDataDescription;
```

Comment:
Align with TD draft, name it `inputData` and `outputData`?

Reply:
`inputData` would imply this is data, but it is a description of input data.